sw/vendor/patches/riscv_compliance/0004-otp-img-Add-OTP-image-preload-switch-to-RUN_TARGET.patch

```diff
@@ -0,0 +1,19 @@
+From 40736d1037ec4c9752927fdbbb4f44b961b51458 Mon Sep 17 00:00:00 2001
+From: Michael Schaffner <msf@google.com>
+Date: Tue, 23 Feb 2021 18:30:20 -0800
+Subject: [PATCH 4/4] [otp-img] Add OTP image preload switch to RUN_TARGET
+
+Signed-off-by: Michael Schaffner <msf@google.com>
+
+diff --git a/riscv-target/opentitan/device/rv32imc/Makefile.include b/riscv-target/opentitan/device/rv32imc/Makefile.include
+index b71105c..079dc5d 100644
+--- a/riscv-target/opentitan/device/rv32imc/Makefile.include
++++ b/riscv-target/opentitan/device/rv32imc/Makefile.include
+@@ -25,6 +25,7 @@ ifeq ($(OT_TARGET),sim_verilator)
+     CARG = -DSIMULATION=1
+     RUN_TARGET = \
+         $(TARGET_SIM) \
++            --meminit="otp,$(OT_BIN)/sw/device/otp_img/otp_img_sim_verilator.elf" \
```

Comment:
just because i know this part sucks to change... is this supposed to be `.vmem`?

Reply:
ah crap, thanks for catching that!
yes it's supposed to be `*.vmem`.

Reply:
fixed!